decidim-generators/lib/decidim/generators/app_templates/sidekiq.yml.erb

```diff
@@ -15,3 +15,4 @@
   - [exports, 1]
   - [close_meeting_reminder, 1]
   - [spam_analysis, 1]
+  - [seeds, 1]
```

Comment:
Is this actually being used by ActiveJob/sidekiq? Shouldn't we migrate at least the call to the seeds through a job and then use this new queue as in a new "queue_as :seeds" or something like this?

Reply:
This is actually coming in a new PR. I will remove it now, so we have a clean merge for the moment.